ScratchPad.py

```diff
@@ -49,5 +49,4 @@
 print(letterlist)
 
 #using list comprehension to do the above task
-letterlist = [i for i in wordlist if letterlist.__contains__(i)==False]
-print(letterlist)
+print([letterlist.append(i) for i in wordlist if i in letterlist])
```

Comment:
Instead of rushing to find a solution that works, try understanding how the list comprehension concept works. it will go a long way in helping you use the concept in the future to solve other problems